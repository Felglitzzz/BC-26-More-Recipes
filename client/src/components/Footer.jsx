import React from 'react';

export default () => {
  return (
    <footer>
      <div>
        <h5>More-Recipes...</h5>
        <p>An Andela Bootcamp LOS/26 project</p>
      </div>
      <div>
        <h5>Project links</h5>
        <ul>
          <li><a href="https://github.com/larrystone/BC-26-More-Recipes">Github</a></li>
          <li><a href="https://www.pivotaltracker.com/n/projects/2096195">Pivotal Tracker</a></li>
        </ul>
      </div>
    </footer >
  )
}