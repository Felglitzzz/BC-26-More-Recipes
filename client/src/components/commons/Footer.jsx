import React from 'react';

/**
 * @description - Stateless component for rendering the foooter
 *
 * @returns {view} FooterView - Rendered view
 */
function FooterView() {
  return (
    <footer
      className="footer"
    >
      <div className="flex-row flex__space-around">
        <div className="footer-item">
          <center>
            <h2>More-Recipes</h2>
            <div>
              <a href="https://github.com/larrystone/BC-26-More-Recipes">
                An Andela project
              </a>
            </div>
            <h5 className="sub-title">
              &copy; {`${new Date().getFullYear()}`} | Lawal Lanre (Larrystone)
            </h5>
          </center>
        </div>
      </div>
    </footer>
  );
}

export default FooterView;
