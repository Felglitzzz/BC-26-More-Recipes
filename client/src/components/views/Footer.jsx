import React from 'react';

const Footer = () => {
  return (
    <footer>
      <div className='flex-row'>
        <div className='footer-item'>
          <h5>More-Recipes...</h5>
          <p>An Andela Bootcamp LOS/26 project</p>
        </div>
        <div className='footer-item'>
          <h5>Project links</h5>
          <ul>
            <li>
              <a href="https://github.com/larrystone/BC-26-More-Recipes">
                Github
              </a>
            </li>
            <li>
              <a href="https://www.pivotaltracker.com/n/projects/2096195">
                Pivotal Tracker
            </a>
            </li>
          </ul>
        </div>
      </div>
      <h6><center>&copy; 2017 Lanre Lawal E. (Larrystone)</center></h6>
    </footer>
  );
};

export default Footer;