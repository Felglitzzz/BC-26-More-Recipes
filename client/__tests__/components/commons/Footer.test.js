import React from 'react';
import { shallow } from 'enzyme';
import Footer from './../../../../client/src/components/commons/Footer';

describe('Component: Footer', () => {
  it('renders properly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
