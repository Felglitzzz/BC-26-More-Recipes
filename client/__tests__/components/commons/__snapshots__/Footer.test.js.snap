// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FooterView />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="flex-row flex__space-around"
      >
        <div
          className="footer-item"
        >
          <center>
            <h2>
              More-Recipes
            </h2>
            <div>
              <a
                href="https://github.com/larrystone/BC-26-More-Recipes"
              >
                An Andela project
              </a>
            </div>
            <h5
              className="sub-title"
            >
              © 
              2018
               | Lawal Lanre (Larrystone)
            </h5>
          </center>
        </div>
      </div>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer-item"
        >
          <center>
            <h2>
              More-Recipes
            </h2>
            <div>
              <a
                href="https://github.com/larrystone/BC-26-More-Recipes"
              >
                An Andela project
              </a>
            </div>
            <h5
              className="sub-title"
            >
              © 
              2018
               | Lawal Lanre (Larrystone)
            </h5>
          </center>
        </div>,
        "className": "flex-row flex__space-around",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <center>
            <h2>
              More-Recipes
            </h2>
            <div>
              <a
                href="https://github.com/larrystone/BC-26-More-Recipes"
              >
                An Andela project
              </a>
            </div>
            <h5
              className="sub-title"
            >
              © 
              2018
               | Lawal Lanre (Larrystone)
            </h5>
          </center>,
          "className": "footer-item",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                More-Recipes
              </h2>,
              <div>
                <a
                  href="https://github.com/larrystone/BC-26-More-Recipes"
                >
                  An Andela project
                </a>
              </div>,
              <h5
                className="sub-title"
              >
                © 
                2018
                 | Lawal Lanre (Larrystone)
              </h5>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "More-Recipes",
              },
              "ref": null,
              "rendered": "More-Recipes",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://github.com/larrystone/BC-26-More-Recipes"
                >
                  An Andela project
                </a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "An Andela project",
                  "href": "https://github.com/larrystone/BC-26-More-Recipes",
                },
                "ref": null,
                "rendered": "An Andela project",
                "type": "a",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "© ",
                  "2018",
                  " | Lawal Lanre (Larrystone)",
                ],
                "className": "sub-title",
              },
              "ref": null,
              "rendered": Array [
                "© ",
                "2018",
                " | Lawal Lanre (Larrystone)",
              ],
              "type": "h5",
            },
          ],
          "type": "center",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="flex-row flex__space-around"
        >
          <div
            className="footer-item"
          >
            <center>
              <h2>
                More-Recipes
              </h2>
              <div>
                <a
                  href="https://github.com/larrystone/BC-26-More-Recipes"
                >
                  An Andela project
                </a>
              </div>
              <h5
                className="sub-title"
              >
                © 
                2018
                 | Lawal Lanre (Larrystone)
              </h5>
            </center>
          </div>
        </div>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="footer-item"
          >
            <center>
              <h2>
                More-Recipes
              </h2>
              <div>
                <a
                  href="https://github.com/larrystone/BC-26-More-Recipes"
                >
                  An Andela project
                </a>
              </div>
              <h5
                className="sub-title"
              >
                © 
                2018
                 | Lawal Lanre (Larrystone)
              </h5>
            </center>
          </div>,
          "className": "flex-row flex__space-around",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <center>
              <h2>
                More-Recipes
              </h2>
              <div>
                <a
                  href="https://github.com/larrystone/BC-26-More-Recipes"
                >
                  An Andela project
                </a>
              </div>
              <h5
                className="sub-title"
              >
                © 
                2018
                 | Lawal Lanre (Larrystone)
              </h5>
            </center>,
            "className": "footer-item",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  More-Recipes
                </h2>,
                <div>
                  <a
                    href="https://github.com/larrystone/BC-26-More-Recipes"
                  >
                    An Andela project
                  </a>
                </div>,
                <h5
                  className="sub-title"
                >
                  © 
                  2018
                   | Lawal Lanre (Larrystone)
                </h5>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "More-Recipes",
                },
                "ref": null,
                "rendered": "More-Recipes",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="https://github.com/larrystone/BC-26-More-Recipes"
                  >
                    An Andela project
                  </a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "An Andela project",
                    "href": "https://github.com/larrystone/BC-26-More-Recipes",
                  },
                  "ref": null,
                  "rendered": "An Andela project",
                  "type": "a",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "© ",
                    "2018",
                    " | Lawal Lanre (Larrystone)",
                  ],
                  "className": "sub-title",
                },
                "ref": null,
                "rendered": Array [
                  "© ",
                  "2018",
                  " | Lawal Lanre (Larrystone)",
                ],
                "type": "h5",
              },
            ],
            "type": "center",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
